### PR TITLE
Resubmission-after-Revision

### PR DESCRIPTION
**Pull request by rahul31agrawal**
Hi,
I have added button with link variant & small text, center text & gray text variant to my component library.
Please review it.

Thank
Text Variant
s
Button with link

### CODE REVIEW

File: cssFiles/button.css [hunk 1]
```diff
@@ -179,4 +179,21 @@
...
     margin: 2rem;
+}
+
+/** links **/
+.linkp {
+    text-decoration: none;
+    padding: 1rem 2rem
+}
+
+.link-primaryp {
```
Comment (Koushith) [inline cssFiles/button.css:190]: guess, its a typo.

**Review by Koushith — APPROVED** [review]
LGTM.

**Comment by Koushith** [comment]
Additionally, you can give some breathing space to the side panel. overall it looks good.

File: cssFiles/button.css [hunk 2]
```diff
@@ -179,4 +179,21 @@
     border-radius: 9999px;
     padding: 0;
     margin: 2rem;
+}
+
+/** links **/
+.linkp {
```
Comment (irajbhinde) [inline cssFiles/button.css:185]: You can improve the class naming here for better readability

File: cssFiles/button.css [hunk 3]
```diff
@@ -179,4 +179,21 @@
...
+}
+
+.link-primaryp {
+    background-color: var(--primary-color);
+    border-radius: 0.5rem;
+    color: white;
+}
+
+.linkp:hover {
+    background-color:#002ead;
```
Comment (irajbhinde) [inline cssFiles/button.css]: If you're using following color multiple times, create a variable for it

File: cssFiles/components/Typography/typography.html [hunk 4]
```diff
@@ -2,91 +2,121 @@
...
-          <a href="../Badge/badge.html">Badge</a>
-          <a href="../Button/button.html">Button</a>
-          <a href="../card/card.html">Card</a>
-          <a href="../image/image.html">Images</a>
-          <a href="../input/input.html">Input</a>
-          <a href="../Navigation/navigation.html">Navigation</a>
-          <a href="../Grid/grid.html">Grid</a>
-          <a href="../slider/slider.html">Slider</a>
-          <a href="../Typography/typography.html">Typography</a>
+  <nav class="navigationM containerM">
```
Comment (irajbhinde) [inline cssFiles/components/Typography/typography.html:13]: class naming could improve here as well. I could not understand what M stands for here

**Review by irajbhinde — COMMENTED** [review]
Overall suggestion / feedback : Use class names which are easy to understand, otherwise the revision looks good.

**Pull request merged**